### ISSUE TITLE
Can't reproduce the Quick Draw Example

### ISSUE THREAD

**Issue by maitrungduc1410** [zaidalyafeai/zaidalyafeai.github.io#22]
Hi Zaid,

Thanks for you great works.

I tried your Quick Draw example in the folder `Sketcher` (in this repository), it's quite nice, both of your pre-trained models work great.

But after following your [Google Colab Sketcher](https://colab.research.google.com/github/zaidalyafeai/zaidalyafeai.github.io/blob/master/sketcher/Sketcher.ipynb), I did exactly the same you wrote in that colab. But the model produced by the colab has very poor results. See my pictures below

<img width="909" alt="Screenshot 2019-12-06 at 13 07 55" src="https://user-images.githubusercontent.com/16630193/70297850-95d9e680-182a-11ea-82ce-b487297f4616.png">
<img width="899" alt="Screenshot 2019-12-06 at 13 08 25" src="https://user-images.githubusercontent.com/16630193/70297859-a5592f80-182a-11ea-8812-d2d8de6394c7.png">

I want to ask you whether the models in your `Sketch folder` using same model architecture with your Colab, do you use same dataset to feed your model?. Could you explain how can I train you model to get same results as your pre-train models?

Thank you for your time.

**Comment by zaidalyafeai** [comment]
Hey @maitrungduc1410 , I am using the same model/dataset in colab. You should be able to reproduce the resutlts using that notebook. Make sure that the predictions in the notebook and on the web produce the same output, try a matrix of zeros.

**Comment by maitrungduc1410** [comment]
Hi @zaidalyafeai ,

After looking back I see that I made a really silly mistake. The model I trained is fine. But I forgot to change `class_name` list. That's why when model produce prediction, it maps to wrong class name

Thank you. I'm going to close this issue

**Comment by maitrungduc1410** [comment]
Hi @zaidalyafeai, 

I re open this issue just because I have one more question related to Quick draw example:
The current data set has 100 classes. If I want to increase the number of classes, is there place in the [Google Colab Sketch](https://colab.research.google.com/github/zaidalyafeai/zaidalyafeai.github.io/blob/master/sketcher/Sketcher.ipynb) I need to change, or I just need to add more class in `mini_classes.txt`

Sorry if it's silly question, I'm new in ML.

Thank you

**Comment by zaidalyafeai** [comment]
@maitrungduc1410 , yes just change that file. However, make sure you don't run out of memory.

**Comment by maitrungduc1410** [comment]
> @maitrungduc1410 , yes just change that file. However, make sure you don't run out of memory.

In the Google Colab, in the section `# Define model` I see you add the layer like below:
```python
model.add(layers.Dense(100, activation='softmax')) 
```
Does the number `100` above represents for the number of classes? So I need to change it also, right?

Thank you

**Comment by zaidalyafeai** [comment]
@maitrungduc1410 yes you are right.
